app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from app.errors import (
+    VaccineError,
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    count_friends = None
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | VaccineError:

```

Comment:
The return type hint `str | VaccineError` is incorrect because the method can raise multiple types of errors, not just `VaccineError`. Consider using `str | Exception` or listing all specific exceptions that can be raised.